app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -984,9 +1000,7 @@ protected void deliverError(FormEntryActivity receiver, Exception e) {
     }
 
     private void handleFormLoadCompletion(AndroidFormController fc) {
-        if (GeoUtils.ACTION_CHECK_GPS_ENABLED.equals(locationRecieverErrorAction)) {
```

Comment:
so earlier the gps error will only show up if the receiver was fired before the form load completes ?

Reply:
Yeah.